docs/source/index.html.haml

```diff
@@ -55,6 +55,17 @@ subtitle: Turn any <code>&lt;input&gt;</code> into an address autocomplete
           %figure.media-object
             %img{src: "https://res.cloudinary.com/hilnmyskv/image/upload/f_auto,w_600/v1462178462/Algolia_Places_-_demo_map_v2_tbsfvm.gif", class: "gif"}
 
+      .row
+        .bloc-5
+          %h3.title autocomplete.js & instantsearch.js ready
+          %p Create full search experiences with autocomplete.js & instantsearch.js plugins for Algolia Places.
```

Comment:
`s/Create full search experiences/Build unique search experiences/` to fit our tagline?